openrtb/macrosubs_test.go

```diff
@@ -6,106 +6,98 @@ import (
 	"github.com/Vungle/vungo/openrtb"
 )
 
-// verifyMacroSubs runs openrtb.MacroSubs and checks if the output is the expected string.
-func verifyMacroSubs(t *testing.T, input string, bidRes *openrtb.BidResponse, expectedOutput string) {
-	result, err := openrtb.MacroSubs(input, bidRes)
-	if err != nil {
-		t.Fatal("MacroSubs should complete successfully")
-	}
+type auctionResult struct {
```

Comment:
FYI, you can do `type auctionResult float64` directly based on your usage. Either way works.